### PR TITLE
Python 12 2019

### PR DESCRIPTION
**Pull request by adrianhall**

### CODE REVIEW

File: releases/2019-12/2019-12-python.md [hunk 1]
```diff
@@ -12,9 +12,18 @@ The Azure SDK team is pleased to make available the December 2019 client library
 
 This release includes the following:
 
+#### Generally Available
+
+- Key Vault (Keys and Secrets)
```
Comment (kaerm) [inline releases/2019-12/2019-12-python.md]: and Certificates (I think)

File: releases/2019-12/2019-12-python.md [hunk 2]
```diff
@@ -12,9 +12,18 @@ The Azure SDK team is pleased to make available the December 2019 client library
...
+- Storage (Blobs, Files, and Queues)
```
Comment (kaerm) [inline releases/2019-12/2019-12-python.md]: File shares

File: releases/2019-12/2019-12-python.md [hunk 3]
```diff
@@ -12,9 +12,18 @@ The Azure SDK team is pleased to make available the December 2019 client library
...
+
 #### Preview
 
 - App Configuration
+- Cosmos
+- Storage (DataLake Files)
```
Comment (kaerm) [inline releases/2019-12/2019-12-python.md]: Just datalake

File: releases/2019-12/2019-12-python.md (hunk 1 above)
Comment (adrianhall) [reply releases/2019-12/2019-12-python.md]: ... nope - checked the release and Certs is preview.

**Pull request merged**